app/scripts/controllers/transactions/index.js

```diff
@@ -582,7 +582,7 @@ class TransactionController extends EventEmitter {
     ].find(tokenMethodName => tokenMethodName === name && name.toLowerCase())
 
     let result
-    let code
+    let code = 'default'
```

Comment:
will this have issues when calculating gas in tx-gas-utils?

Reply:
for simple sends

Reply:
good catch, I think so...

Reply:
@frankiebee What do you think of the approach in the latest commit? aaca727d7349adb6d3f321582865467dd0a15cde